guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -467,6 +467,13 @@ You can invoke `test_jdbcmysql`, `test_jdbcsqlite3` or `test_jdbcpostgresql` als
 
 The test suite runs with warnings enabled. Ideally, Ruby on Rails should issue no warnings, but there may be a few, as well as some from third-party libraries. Please ignore (or fix!) them, if any, and submit patches that do not issue new warnings.
 
+### Updating the documentation
+The Ruby on Rails [guides](https://guides.rubyonrails.org/) are designed to make developers immediately productive with Rails, and to help them understand how all of the pieces fit together.
```

Comment:
There is a tendency to cram things into the guides that might be better suited to the API docs.  Perhaps we can nudge contributors in the right direction:

```suggestion
The Ruby on Rails [guides](https://guides.rubyonrails.org/) provide a high-level overview of Rails' features, while the [API documentation](https://api.rubyonrails.org/) delves into specifics.
```